src/index.html

```diff
@@ -11,6 +11,42 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    action="#"
+    method="post"
+    data-qa="big"
+    class="search big-search"
+    >
+      <label class="container big-container">
+        <img src="./images/Search.svg"
+       alt="icon"
+       class="image big-images"
+      >
+        <input
+        type="text"
+        placeholder="Try &quot;Los Angeles&quot;"
+        data-qa="keypress"
+        class="input big-input"
+      >
+      </label>
+    </form>
+    <form
+    action="#"
+    method="post"
+    data-qa="small"
+    class="search search-small"
+  >
+      <label class="container small-cont">
```

Comment:
each level of nesting should have exactly 2 indents
```suggestion
    <label class="container small-cont">
```